codespell_lib/data/dictionary.txt

```diff
@@ -29677,6 +29690,9 @@ splitted->split
 splittng->splitting
 spllitting->splitting
 spoace->space
+spoaced->spaced
+spoaces->spaces
+spoacing->spacing
```

Comment:
I did wonder about poach... but probably unlikely.

Reply:
Yes, indeed this looks to me quite unlikely so hope it is o.K. if i'm resolving this conversation now.